index.bs

```diff
@@ -369,6 +448,110 @@ Report long tasks {#report-long-tasks}
         1. [=Queue the PerformanceEntry=] |newEntry|.
 </div>
 
+
+Frame Timing Info {#sec-frame-timing-info}
+------------------------------------------
+<dfn export>frame timing info</dfn> is a [=struct=] used as a bookkeeping detail by the long animation frame algorithms.
+It has the following [=struct/items=]:
+
+<dl dfn-for="frame timing info">
+    : <dfn>start time</dfn>
```

Comment:
Should we initialize all these?

Reply:
We are. a single `dd` for multiple `dt` is ok

Reply:
oh ok, TIL!